### DISCUSSION TITLE
Timeline for Ubuntu 22.04 content

### DISCUSSION THREAD

**Discussion by alexhaydock** [ComplianceAsCode/content#8465] [General]
Hi!

I've noticed there's been quite an uptick of Ubuntu contributions to CaC over the past year, including the release of the first CIS content for Ubuntu 20.04. I'm just wondering whether this represents a shift in how Ubuntu content will be worked on in CaC going forward.

Mainly I'm wondering whether work will start on new content for 22.04 in CaC quicker than with 20.04 where, as an example, the CIS content didn't ship until CaC 0.1.57 in Jul 2021.
 
Taking CIS content as an example, Eric Pinnell of CIS commented recently on a CIS WorkBench thread:

> With The Ubuntu LTS releases, we attempt to have a new Benchmark available within 90 days of the official release date. We're tracking on 22.04 being releases in April, with a Benchmark release date toward the end of July

I can see contributions from Canonical employees and I think I'm right in thinking that CaC is now being treated as downstream of the internal [Ubuntu Advantage code](https://ubuntu.com/security/cis), which seems to be in contrast to the RHEL content, where there's been a lot of public work in CaC for RHEL9 already.

Essentially what I'm wondering is whether anyone knows whether things are changing with the Ubuntu content - will we be able to see and contribute to development soon? Or should I expect the pattern to be similar to 20.04 where content is worked on for UA internally and then starts to appear in CaC mid 2023?

Apologies if I've misunderstood how the contributions from Canonical are working here! I'm grateful to all the content that's made available and looking forward to being able to contribute to 22.04 content if I can. :smile:

**Comment by dodys** [comment]
Hi Alex, we currently don't have any timeline as we are expecting for the document drafts to be created/released.
I will try to keep this topic up-to-date if we have any news.

**Reply by alexhaydock** [reply]
Looks like this has been done as part of #9953! Great work all :smiley:

**Reply by dodys** [reply]
do note that it is not yet finalized, but the base is there :)
